src/scripts/components/pages/Signup.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import SignupFormContainer from 'containers/SignupFormContainer'
+import Anchor from 'components/foundations/buttons/Anchor'
+import Card from 'components/structures/Card'
+
+const FooterText = styled.p`
+  font-size: ${props => props.theme.fonts.text.small};
+`
+const NavLink = Anchor.withComponent(Link)
+
+class Signup extends Component {
+  render () {
+    return (
+      <div>
+        <Card
+          title="Sign up"
+          footer={
+            <FooterText>
+              Already have an account?{' '}
```

Comment:
Same do we need this `{' '}`? Could it be taken care of w/ css?

Reply:
Removing this...